setup_git_crypt.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+pushd /tmp
+wget https://github.com/AGWA/git-crypt/archive/master.zip -O git-crypt.zip
```

Comment:
Should we use a specific release instead of the master branch?

This looks like the latest
https://github.com/AGWA/git-crypt/archive/0.5.0.zip

Reply:
I haven't had any issues with master.  Also the 0.5.0 is pretty old, was released on May 30, 2015

Reply:
Ok, then should we use https://github.com/AGWA/git-crypt/archive/ccdcc76f8e1a639847a8accd801f5a284194e43f.zip

The issue with using a branch is the artifact is generally expected to change over time. Locking the artifact to a specific version via commit makes the build more reproducible.